.github/workflows/pytest-cpu.yaml

```diff
@@ -1,48 +1,47 @@
-name: Pytest-CPU
+name: Pytest CPU
 on:
-  pull_request:
-    branches:
-      - dev
-      - main
-      - release/**
   workflow_call:
-  workflow_dispatch:
-# Cancel old runs when a new commit is pushed to the same branch if not on main or dev
-concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: ${{ github.ref != 'refs/heads/main' && github.ref != 'refs/heads/dev' }}
+    inputs:
+      container:
+        required: true
+        type: string
+      name:
+        required: true
+        type: string
+      pytest-command:
+        required: true
+        type: string
+      pytest-markers:
+        required: true
+        type: string
+      pytest-s3-bucket:
+        required: false
+        type: string
+      python-version:
```

Comment:
Is this used anywhere?

Reply:
Good catch, it's not but it probably should be.  Fix coming.